src/main/views/courts/courts.njk

```diff
@@ -1,31 +1,99 @@
-{% extends "template.njk" %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
+{% extends "template.njk" %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Courts and Tribunals</h1>
-
-  <table class="govuk-table" id="courts">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Name</th>
-        <th scope="col" class="govuk-table__header">Last Updated</th>
-        <th scope="col" class="govuk-table__header"></th>
-        <th scope="col" class="govuk-table__header"></th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-    {% for court in courts | sort(attribute="name") %}
-      <tr class="govuk-table__row">
-        <td scope="row" class="govuk-table__cell">{{ court.name }}</td>
-        <td scope="row" class="govuk-table__cell">{{ court['updated_at'] }}</td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="view-{{ court.slug }}" class="govuk-link" href="{{ factFrontendURL }}/courts/{{ court.slug }}">view</a>
-        </td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="edit-{{ court.slug }}" class="govuk-link" href="/courts/{{ court.slug }}/edit#general">edit</a>
-        </td>
-      </tr>
+  <h1 class="govuk-heading-xl">Courts and tribunals</h1>
+
+  <div id = 'tableContainer'>
+    <input type='hidden' name='courtsUpdatedAscToggle' id='courtsUpdatedAscToggle' value='INACTIVE' />
+    <input type='hidden' name='courtsNameAscToggle' id='courtsNameAscToggle' value='ASC' />
+
+    <div id='courtOptionsRow' class='govuk-grid-row'>
+      <div class='govuk-grid-column-one-half'>
+        {{ govukInput({
+          attributes: {
+            placeholder: ' Search courts and tribunals'
+          },
+          prefix: {
+            text: '🔎'
```

Comment:
Is there a gov-uk example we can use? I'm not sure if such a thing exists in the GDS framework.

Reply:
Good question, I know Pauline was happy with it as is, but I can't see anything either in the toolkit that allows us to bring across icons to use, probably me being silly there. Do you want me to spend some more time looking into it? Would it could be an accessibility issue?